skeleton/backend/src/Backend.hs

```diff
@@ -4,8 +4,12 @@
 module Backend where
 
 import Common.Route
+import Frontend
 import Obelisk.Backend
 
+backendMain :: IO ()
+backendMain = runBackend backend frontend
```

Comment:
I don't feel like moving this here adds anything. Both before and after, it was part of the skeleton.

Reply:
Well, it makes ob repl/watch/run typecheck everything except the trivial `main = backendMain`. Doesn't matter for `ob init` but It's pretty annoying to have a full build fail on this one line rather than at ghci(d) if/when this part of the skeleton changes, which is the main point of #424